content/techknowlogick/drone-b2/index.md

```diff
@@ -0,0 +1,90 @@
+---
+date: 2017-12-04T00:00:00+00:00
+title: Backblaze B2
+author: techknowlogick
+tags: [ backblaze, b2, storage ]
+repo: techknowlogick/drone-b2
+logo: backblaze-b2-logo.svg
+image: techknowlogick/drone-b2
+---
+
+The B2 plugin uploads files and build artifacts to your B2 bucket. The below pipeline configuration demonstrates simple usage:
+
+```yaml
+pipeline:
+  b2:
+    image: techknowlogick/drone-b2
+    bucket: my-bucket-name
+    b2_account_id: a50d28f4dd477bc184fbd10b376de753
```

Comment:
This example won't work. Every parameter defined like that will be prefixed with plugin_ and gets injected as an environment variable, but you are not listening for this env variable.